heron/api/src/scala/com/twitter/heron/streamlet/scala/converter/ScalaToJavaConverter.scala

```diff
@@ -27,6 +32,17 @@ object ScalaToJavaConverter {
       override def get(): T = f()
     }
 
+
+  def toJavaSource[T](source: Source[T]): com.twitter.heron.streamlet.Source[T] = {
```

Comment:
suggest to use import alias.

import com.twitter.heron.streamlet.{Source => JavaSource}

Reply:
I will leave it as is, this is a nice to have that I can put in once we make deeper changes, right now lets focus on getting Builder operational